### PR TITLE
Zip code search v1

### PR DESCRIPTION
**Pull request by mager**
screen shot 2016-03-02 at 1 58 25 pm

## TODO

* Add label when zipcode boundary is created
* Remove last zip code when hitting submit?
* Rewrite in React/Redux

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -28,6 +28,28 @@ app.get('/zones', function(req, res) {
...
+app.get('/zips', function(req, res) {
+
+  var base_url = 'https://vanitysoft-boundaries-io-v1.p.mashape.com/reaperfire/rest/v1/public/boundary';
+  var query_params = '?includepostal=true&limit=1&zipcode=';
+  var zip = req.query.zip;
+
+  var options = {}
+  options.url = `${base_url}${query_params}${zip}`;
+  options.headers = {
+    'X-Mashape-Key': process.env.MASHAPE_KEY
```
Comment (drdelambre) [inline app.js]: nice

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon app.js"
```
Comment (drdelambre) [inline package.json]: will this mess up heroku? might want to make it `start:dev`

File: public/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        fillColor: '#7B858B'
+      });
+    })
+    .addTo(map);
+}
+
+function center_map(q) {
+    var base_url = 'https://api.mapbox.com/geocoding/v5/mapbox.places/';
+    var end_url = '.json?access_token=';
+    var request_url = `${base_url}${q}${end_url}${MAPBOX_TOKEN}`;
```
Comment (drdelambre) [inline public/scripts/main.js]: fancy! you should set up a transpiler if you wanna use this stuff and start penetrating older browsers

**Comment by drdelambre** [comment]
it's progressing nicely. gotta work on that state thing next

File: package.json (hunk 2 above)
Comment (mager) [inline package.json]: @drdelambre I think Heroku uses the Procfile instead.

File: public/scripts/main.js (hunk 3 above)
Comment (mager) [inline public/scripts/main.js]: I wanna rewrite the whole thing in React + Redux.

**Comment by mager** [comment]
@drdelambre r?

**Pull request closed without merging**